contracts.proto

```diff
@@ -149,6 +150,10 @@ message Statement {
     TypeDescriptor typeDescriptor = 1;
     repeated ErrorCode errorCodes = 2;
   }
+  message Operation {
+    TypeDescriptor typeDescriptor = 1;
+    TypeRef returnType = 2;
+  }
```

Comment:
Are operations now officially a thing and clients should implement support for them?

Reply:
They are, although it is not mandatory (yet) for clients to support them.